### PR TITLE
Fix Quicksand cave error and fix rank 2-3 san d'oria

### PR DESCRIPTION
**Pull request by Chronos77**

### CODE REVIEW

File: scripts/zones/Chateau_dOraguille/npcs/Halver.lua [hunk 1]
```diff
@@ -102,7 +102,7 @@ function onTrigger(player,npc)
 		-- Mission San D'Oria 2-3 Journey Abroad 
 		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 11) then
 			player:startEvent(0x01fb);
-		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 0) then
+		elseif(currentMission == JOURNEY_ABROAD and MissionStatus == 1) then
```
Comment (TeoTwawki) [inline scripts/zones/Chateau_dOraguille/npcs/Halver.lua]: If someone was mid mission when he got changed from 1 to 0 he will be broken for that player. Likewise, this change breaks him for the next batch of people.
https://github.com/DarkstarProject/darkstar/pull/1247
He's been changed back and forth multiple times from ppl "fixing" him at this exact spot.

Comment (TeoTwawki) [inline scripts/zones/Chateau_dOraguille/npcs/Halver.lua]: To fix var for players that started a quest or mission that got changed while they were in middle of it, use 
```
@setplayervar <player name> <variable name> <value>
```

Comment (Chronos77) [inline scripts/zones/Chateau_dOraguille/npcs/Halver.lua]: Ok thanks ! i remove that for this pull request

Comment (Turttle) [inline scripts/zones/Chateau_dOraguille/npcs/Halver.lua]: y u poke smelly elvaan *sadface*

**Pull request merged**